docs: add a note about setting file executability (#2435)
defenseunicorns/zarf: site/src/content/docs/ref/components.mdx

- Relative paths to either a file or directory (from the `zarf.yaml` file)
- A remote URL (http/https)
- Verified using the `shasum` field for data integrity (optional and only available for files)

> - made executable by setting `Executable` boolean value [true|false]

Reviewer: Small suggestion on capitalization consistency and wording

```suggestion
- Made executable by setting the `executable` field to `true`
```
